app-containers/containers-common/containers-common-0.56.0.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Common config files and docs for Containers eco-system"
+
+HOMEPAGE="https://github.com/containers/common"
+
+LICENSE="Apache-2.0"
+
+SLOT="0"
+
+DEPEND="
+	>=dev-go/go-md2man-2.0.2
+"
+
+RDEPEND="
+	app-containers/containers-image
+	app-containers/containers-storage
+	app-containers/containers-shortnames
+	net-firewall/nftables
+	net-firewall/iptables[nftables]
+	|| ( app-containers/crun app-containers/runc )
+	|| (
+		( >=app-containers/netavark-1.6.0 >=app-containers/aardvark-dns-1.6.0 )
+		>=app-containers/cni-plugins-0.9.1
+	)
+"
+
+is_live() {
+	[[ ${PV} == *9999* ]]
+}
+
+if is_live; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/common.git"
+else
+	SRC_URI="https://github.com/containers/common/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~arm-linux ~arm64-linux ~ppc64-linux ~riscv-linux ~x86-linux ~arm64-macos ~ppc-macos ~x64-macos ~x64-solaris"
+fi
+
+is_live || S="${WORKDIR}/${P#containers-}"
+
+src_prepare() {
+	default
+
+	[[ -f docs/Makefile ]] || die
+	sed -i -e 's|/usr/local|/usr|g' docs/Makefile
+
+	cat <<"EOF" > "${T}/fix-warnings.patch"
```

Comment:
Can't you put this in a proper patch in files/ and then add it to PATCHES in the ebuild?